app/src/main/java/com/codepath/apps/critterfinder/fragments/SwipeablePetsFragment.java

```diff
@@ -13,6 +14,7 @@
 import android.widget.TextView;
 
 import com.codepath.apps.critterfinder.R;
+import com.codepath.apps.critterfinder.activities.PetSearchFilterActivity;
```

Comment:
this import looks funny to me. is it unused and if so can we delete it? The swipeable pets fragment shouldn't know anything about the pet search filter activity.